example-nextjs/pages/index.tsx

```diff
@@ -0,0 +1,20 @@
+import Head from "next/head";
+import "@worldcoin/idkit/build/index.css";
```

Comment:
This shouldn't be required, as the styles get loaded automatically by the widget.